Error: Cannot find module '../../third-party/specifications'

Is the dev branch currently broken? I just wanted to give the server a quick try after reading about it on community.particle.io, but I'm getting the following error message (see below). Am I doing something wrong? 

```
npm start

> spark-server@0.1.1 start /Users/Test/particle.io/spark-server
> nodemon --exec babel-node ./src/main.js --watch src --watch ../spark-protocol/dist --ignore data

[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: /Users/Test/particle.io/spark-server/src/**/* ../spark-protocol/dist
[nodemon] starting `babel-node ./src/main.js`
module.js:472
    throw err;
    ^

Error: Cannot find module '../../third-party/specifications'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/Test/particle.io/spark-server/node_modules/spark-protocol/dist/lib/FirmwareManager.js:67:23)
    at Module._compile (module.js:571:32)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/Test/particle.io/spark-server/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/Test/particle.io/spark-server/node_modules/spark-protocol/dist/server/DeviceServer.js:63:24)
    at Module._compile (module.js:571:32)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/Test/particle.io/spark-server/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/Test/particle.io/spark-server/node_modules/spark-protocol/dist/index.js:24:21)
    at Module._compile (module.js:571:32)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/Test/particle.io/spark-server/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/Test/particle.io/spark-server/src/defaultBindings.js:5:1)
    at Module._compile (module.js:571:32)
    at loader (/Users/Test/particle.io/spark-server/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/Test/particle.io/spark-server/node_modules/babel-register/lib/node.js:154:7)
[nodemon] app crashed - waiting for file changes before starting...

```

--- a/src/scripts/update-firmware-binaries.js
+++ b/src/scripts/update-firmware-binaries.js
@@ -214,7 +214,7 @@ const downloadAppBinaries = async (): Promise<*> => {
 
   const specificationsResponse = await githubAPI.repos.getContent({
     owner: GITHUB_USER,
-    path: 'lib/deviceSpecs/specifications.js',
+    path: 'oldlib/deviceSpecs/specifications.js',
     repo: GITHUB_CLI_REPOSITORY
   });
 

